authorization/views.py

```diff
@@ -67,7 +68,7 @@ def handle_no_permission(self):
 
         # For authenticated users, raise exception.
```

Comment:
"raise exception" in the comment is not correct anymore.

Is the response status code now 200 instead of 403? Then, the code wouldn't describe the real condition. I wonder if we could keep the 403 status and still avoid the X-Frame-Options DENY response header.

But this isn't very critical if you can't find simple solutions.